ra2ce/graph/networks.py

```diff
@@ -80,8 +81,22 @@ def __init__(self, config, files):
             self.region_var = None
 
         # Cleanup
-        self.snapping = config["cleanup"]["snapping_threshold"]
-        self.segmentation_length = config["cleanup"]["segmentation_length"]
+        self._cleanup_options = config.get("cleanup", {})
+        if bool(self._cleanup_options) and any(
+            [v for k, v in self._cleanup_options.items()]
+        ):
+            self._cleanup_options.update({"cleanup": True})
```

Comment:
Not sure what is the idea of this, are you just trying to verify if there are entries in the dictionary?
```suggestion
        if any(_cleanup_options.items):
            _cleanup_options["cleanup"] = True
```
I do not think this is necessary though, the user should explicitly specify whether to cleanup or not.

Reply:
The idea is that if the user have specified all the options as either None or False, we should not call the cleanup functions. But I guess later once the cleanup function is ready, the logic will be handled there correctly.